lib/DDG/Rewrite.pm

```diff
@@ -159,9 +159,9 @@ sub _build_nginx_conf {
 		# the sorted keys into a single string.
 		# e.g. post_body '{"method":"$2","query":"$1","cleaned_query":"$1"}'
 		# Would give a $cache_keys value of '$1$2'
-		my $cache_keys = join '', sort keys {
+		my $cache_keys = join '', sort keys %{ {
```

Comment:
@chorny By the way, what is the error raised when using the origininal version of this?